.gitignore

```diff
@@ -62,5 +62,4 @@ junit_results.xml
 cppunit_results.xml
 boost_test_results.xml
 **/.objs/*.*
-!**/.objs/.gitignore
-
+!**/.objs/.gitignore
```

Comment:
It's fine to remove the completely blank line, but please add back the newline at the end of the last non-blank line.  Same for `.dockerignore`.